app/views/items/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<center>
+<h2>Update <a href="/items/<%=@item.id%>"><%= @item.name  %></a></h2>
+<%= form_tag "/items/#{@item.id}", method: :patch, :class => "item-edit" do %>
+  Name: <%= text_field_tag :name, @item.name %><br/><br/>
+  Price : <%= text_field_tag :price, @item.price %><br/><br/>
+  <p>Description:</p> <%= text_area_tag :description, @item.description, :id => "desc" %><br/><br/>
+  Image    : <%= text_field_tag :image, @item.image %><br/><br/>
+  Inventory    : <%= text_field_tag :inventory, @item.inventory %><br/><br/>
+  <%= submit_tag "Submit Item Update" %>
+<% end %>
+</center>
```

Comment:
If you want these labels next to each text field, you can also use the label_tag: to do this. It would look something like: 
```html.erb
<%= label_tag: :name %>
<%= text_field_tag: :name, placeholder %>
```